### DISCUSSION TITLE
doris2.1.2 + openjdk17 be启动报错

### DISCUSSION THREAD

**Discussion by a1026431636** [apache/doris#34331] [RFC]
![image](https://github.com/apache/doris/assets/69331902/1ea9fdbc-66c9-4a15-8c48-441ccde6453e)

**Comment by a1026431636** [comment]
start_be.sh: 行 365: 18372 段错误               ${LIMIT:+${LIMIT}} "${DORIS_HOME}/lib/doris_be" "$@" >> "${LOG_DIR}/be.out" 2>&1 < /dev/null
是不兼容openjdk17吗？ BE的配置文件 JAVA_HOME设置成jdk1.8能正常启动

**Reply by yagagagaga** [reply]
2.1.x 版本只支持 JDK1.8，未来的版本会支持JDK17
